context/brand-context/HISTORY.md

```diff
@@ -1,4 +1,7 @@
 # History
+## 21.0.0 (2022-03-21)
```

Comment:
I wonder if this is more a major breaking change? 

Are we confident that changing things here won't break any existing usage in other project?

Do we need to update any components in the `springernature` toolkits to match this? Or would that be a separate PR?

Reply:
Good point about the components in SN toolkits. I checked all of them and none of them use the SN bespoke breakpoints. I think only SN header used breakpoints and it was looking for default branding breakpoints.

I suppose we could have both version of keys inside `$context--breakpoints` as an interim step before completely removing. Either way it should be fairly easy to update legacy key names to the newer default version. 

TBH, the best thing to do is completely remove SN breakpoints settings file altogether and just refer to the default breakpoints like Springer, Nature do.

Reply:
Let's keep both for now but remove the device specific breakpoints. 

I'm happy to go with this. 

If you can tidy up the conflicts I think we can get this merged.